Sequence.java

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner;
+
+public class Sequence {
+  public static void main(String[] args) {
+    System.out.println("Enter the numbers through a space: ");
+    Scanner in = new Scanner(System.in);
+    String seq = in.nextLine();
+    String seqParts[] = seq.split(" ");
+    int length = seqParts.length;

```

Comment:
do not use temporary variable to store length. Use seqParts.length explicitly on line 11